tests/UI/pages/BO/catalog/discounts/catalogPriceRules/index.js

```diff
@@ -17,7 +17,7 @@ class CatalogPriceRules extends BOBasePage {
     this.pageTitle = 'Catalog Price Rules •';
 
     // Selectors header
-    this.addNewCatalogPriceRuleButton = '#page-header-desc-specific_price_rule-new_specific_price_rule';
+    this.addNewCatalogPriceRuleButton = 'a[data-role="page-header-desc-specific_price_rule"]';
```

Comment:
```suggestion
    this.addNewCatalogPriceRuleButton = 'a[data-role=page-header-desc-specific_price_rule]';
```

You don't need the double quote, you can delete it everywhere

Reply:
Donc thx :+1: